ui/pages/onboarding-flow/add-network-modal/index.js

```diff
@@ -24,13 +23,13 @@ export default function AddNetworkModal() {
   return (
     <>
       <Box paddingTop={4}>
-        <Typography
-          variant={TypographyVariant.H4}
+        <Text
+          variant={TextVariant.headingSm}
           align={TEXT_ALIGN.CENTER}
-          fontWeight={FONT_WEIGHT.BOLD}
+          fontWeight="bold"
```

Comment:
Should use the enum `FontWeight.Bold` here or be able to discern that `headingSm` is bold
 ```suggestion
          fontWeight={FontWeight.Bold}
```